html/scss/navbar.scss

```diff
@@ -0,0 +1,20 @@
+.navbar {
+  position: fixed;
+  width: 100%;
+  height: $nav-height;
+  background: $white;
+  box-shadow: 0 5px 20px $grey;
+  z-index: 999;
+}
+
+.nav-wolox {
+  position: absolute;
```

Comment:
you can use display flex in the `.navbar` element and space between to separate the image from the text